internal/envoy/v3/cluster.go

```diff
@@ -177,6 +179,7 @@ func ExtensionCluster(ext *dag.ExtensionCluster) *envoy_cluster_v3.Cluster {
 				ext.UpstreamValidation,
 				ext.SNI,
 				ext.ClientCertificate,
+				ext.UpstreamTLS,
```

Comment:
Is this set anywhere? I believe it should be set here https://github.com/KauzClay/contour/blob/5233293fc71feaf8461e619d6ca12257f46c78de/internal/dag/extension_processor.go#L103-L117